packages/example_hardhat/package.json

```diff
@@ -2,37 +2,58 @@
   "name": "@sandbox-smart-contracts/example_hardhat",
   "version": "0.0.1",
   "description": "Template package - Hardhat",
+  "files": [
+    "contracts"
+  ],
   "scripts": {
+    "lint": "eslint --max-warnings 0 \"**/*.{js,ts}\" && solhint --max-warnings 0 \"contracts/**/*.sol\"",
+    "lint:fix": "eslint --fix \"**/*.{js,ts}\" && solhint --fix \"contracts/**/*.sol\"",
+    "format": "prettier --check \"**/*.{ts,js,sol}\"",
+    "format:fix": "prettier --write \"**/*.{ts,js,sol}\"",
+    "test": "hardhat test",
     "coverage": "hardhat coverage --testfiles 'test/*.ts''test/*.js'",
-    "test": "hardhat test"
+    "hardhat": "hardhat",
+    "compile": "hardhat compile"
+  },
+  "mocha": {
+    "require": "hardhat/register",
+    "timeout": 40000,
+    "_": [
+      "test/**/*.ts"
+    ]
   },
   "devDependencies": {
     "@ethersproject/abi": "^5.7.0",
     "@ethersproject/providers": "^5.7.2",
-    "@nomicfoundation/hardhat-chai-matchers": "^1.0.6",
-    "@nomicfoundation/hardhat-network-helpers": "^1.0.0",
+    "@nomicfoundation/hardhat-chai-matchers": "1",
+    "@nomicfoundation/hardhat-network-helpers": "^1.0.8",
     "@nomicfoundation/hardhat-toolbox": "^2.0.2",
-    "@nomiclabs/hardhat-ethers": "npm:hardhat-deploy-ethers@^0.3.0-beta.13",
+    "@nomiclabs/hardhat-ethers": "^2.2.3",
     "@nomiclabs/hardhat-etherscan": "^3.1.7",
-    "@openzeppelin/contracts": "^4.8.2",
-    "@openzeppelin/contracts-upgradeable": "^4.9.0",
-    "@typechain/ethers-v5": "^10.2.1",
-    "@typechain/hardhat": "^6.1.6",
+    "@typechain/ethers-v5": "^10.1.0",
+    "@typechain/hardhat": "^6.1.2",
     "@types/chai": "^4.3.5",
     "@types/mocha": "^10.0.1",
-    "@types/node": "^20.1.2",
+    "@types/node": "^20.2.5",
+    "@typescript-eslint/eslint-plugin": "^5.59.8",
+    "@typescript-eslint/parser": "^5.59.8",
     "chai": "^4.3.7",
-    "dotenv": "^16.1.4",
-    "ethers": "^5.7.2",
+    "dotenv": "^16.1.3",
```

Comment:
Why downgrade a bunch of stuff?
And why remove OZ?

Reply:
OZ is not mandatory. 
Regarding the versions: last time I checked hardhat toolbox was choosing some specific versions of ethers, we can try to upgrade packages one by one to see if everything keep working. Specifically I think the minimal project doesn't need dotenv.

Reply:
I would like OZ in the template because ppl are getting confused about how to add deveDependencies vs dependencies and run the new monorepo